src/html/parse-markdown.js

```diff
@@ -19,7 +19,8 @@ function parseMarkdown(context, action) {
   const body = setdefault(content, 'body', '');
 
   const request = setdefault(context, 'request', {});
```

Comment:
I'm not sure I like either variant. Both have side effects (modify context.request) that are hidden in the parse-markdown step. Would there be any harm in simply extracting the extension as a local const?

Reply:
`modify context.request` was already there before. I did not change that part.
Pb is that there is no extension set, thus it has to be set somewhere anyway. And before previous refactoring, it was set in the `make-html` (not better, I agree).

I started to refactor the whole thing (extract those default setters and put them before). And then reverted the changes because all the tests were failing and this was a huge change that would have a big impact. Out of scope for this PR. I would be a topic for another issue. I would fix the current real issue we have here and create a follow up issue for the refactoring.

Reply:
Just a comment in general, your PR is fine to be merged in my eyes.

Reply:
Also, we coerce defaults for secrets using the schema and could do the same for other context values.

Reply:
I think this is completely at the wrong place. IMO action and context objects should be setup before.